.github/dependabot.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/client"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 3
+
+  - package-ecosystem: "cargo"
+    directory: "/communication"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 3
+
+  - package-ecosystem: "cargo"
+    directory: "/engine"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 3
+
+  - package-ecosystem: "cargo"
+    directory: "/client"
```

Comment:
Why is it in here twice?

Reply:
Whoops, must have been a copy-paste mistake